### DISCUSSION TITLE
Unable to write to log fil on synology

### DISCUSSION THREAD

**Discussion by oliverbayer** [dani-garcia/vaultwarden#4948] [Q&A]
Hi, 
I'm running vaultwarden on a synology NAS (Docker container) and everything is working weill, except, that the log files are not beeing created. I set the env.variables: 
LOG_FILE: /path/
ReadWritePaths: /path/

but the file is not beeing created. 
Does anybody knows what could be wrong?
Thanks
Oliver

**Comment by stefan0xC** [comment]
The folder `/path` is not mounted to the container so it's not visible. And secondly you want to specify the file name. So I'd change it to `LOG_FILE: /data/vaultwarden.log` (so e.g. if you mounted `/var/lib/vaultwarden` to `/data` the log file will be created at `/var/lib/vaultwarden/vaultwarden.log`).

Regarding `ReadWritePaths:` isn't that more of a [feature for systemd unit files](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#ReadWritePaths=)?

**Comment by Gerardv514** [comment]
Works for me, this is my variable. 

![IMG_7510](https://github.com/user-attachments/assets/58053efe-da2e-46dc-a0d5-af41fd666053)
